.github/workflows/update-version.yaml

```diff
@@ -74,3 +74,28 @@ jobs:
         git status
         git diff
         exit 1
+    - name: "Switch to master branch"
```

Comment:
The code change snippet won't run if execution fails at Line 72. But AFAIK, that step will die only when having a lower version. It would be better if we could create a separate job for the master branch. That way, it will look good in the GitHub action flows and be less error prone. We can use the previous job's calculation easily in the next job.

Reply:
I thought about this while testing this change, we could use continue on error, however it is better to actually fail if any step fails, and I am not sure if a separate action would also accomplish it. 

I guess since we have tested that this works we can go ahead and merge it and potentially refactor it in a future change.